src/icon/variables.scss

```diff
@@ -1,6 +1,5 @@
 // Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 // See LICENSE.md in the project root for license terms and full copyright notice.
-@import '../style/index';
 
 /// Sizes
 
```

Comment:
Issue says review icons code too... We dont want to do anything with the mixins part? Just having an icon size variable is not that useful, cause you will need to write something like `width: $iui-icons-default;
  height: $iui-icons-default;`.
  But if there was a mixin, you could just include it and forget.. So not sure. Just those current mixins include colors too, which might not be that needed. 
  If we remove colors from mixins, then is a little bit bigger refactor (we could replace all instances of ours of width and height with just the mixin).